frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -235,10 +236,20 @@ impl TypeDartGeneratorTrait for TypeEnumRefGenerator<'_> {
                 .variants()
                 .iter()
                 .map(|variant| {
+                    let variant_name = if self.context.config.dart_enums_style {
+                        if crate::utils::check_for_keywords(&[variant.name.dart_style()]).is_err() {
+                            variant.name.dart_style().to_case(Case::Pascal)
+                        } else {
+                            variant.name.dart_style().to_case(Case::Camel)
+                        }
```

Comment:
uber-nit: maybe sth like:

crate::utils::auto_rename_avoid_dart_keyword_collision(variant.name.dart_style().to_case(Case::Camel))

and

fn auto_rename_avoid_dart_keyword_collision(s: &str) {
  if check_for_keywords { s + "_" } else { s}
}

(pseudocode and pseudo function name only ;) )